syft/execution/plan.py

```diff
@@ -140,6 +141,14 @@ def __init__(
         if not hasattr(self, "forward"):
             self.forward = forward_func or None
 
+        """
+        When we use methods defined in a framework (like: torch.randn) we have a framework
+        wrapper that helps as register and keep track of what methods are called
+        With the below lines, we "register" what frameworks we have support to handle
+        """
```

Comment:
Also, maybe these wrapped_frameworks could go into the tracing file instead?

Reply:
Yeah, I’d rather not have all this in the init for every Plan object.

Reply:
It might make sense to do this in a similar style to the translators, which are registered on the Plan class at the bottom of the file.

Reply:
Doing that now ^_^

Reply:
Can we move this docstring where it belongs? 🙂

Reply:
Ops :\